readme/gt.md

```diff
@@ -0,0 +1,12 @@
+### cmd.gt
+
+| name        | return value  | description   |
+|-------------|---------------|---------------|
+| `gt`       | `[false, true, ...]`  | Returns the result of comparison. |
+
+The following example returns boolean for each value greater than 4:
```

Comment:
Change it to say `returns true for each value greater than 4:` since technically returns booleans all the time :)